csharp/ql/lib/semmle/code/csharp/exprs/Call.qll

```diff
@@ -175,8 +175,8 @@ class Call extends DotNet::Call, Expr, @call {
    * - Line 10: The static target is `Type.InvokeMember()`, whereas the run-time targets
    *   are both `A.M()` and `B.M()`.
    *
-   * - Line 16: There is no static target (delegate call) but the delegate `i => { }` (line
-   *   20) is a run-time target.
+   * - Line 16: There is no static target (delegate call) but the delegate `i => { }`
+   *   (line 20) is a run-time target.
```

Comment:
I don't mind this change, but hopefully it is not needed to generete proper QL doc?

Reply:
It actually is needed.  Here is what the rendered documentation currently looks like, without this fix:

https://codeql.github.com/codeql-standard-libraries/csharp/semmle/code/csharp/exprs/Call.qll/predicate.Call$Call$getARuntimeTarget.0.html
<img width="600" alt="line-20" src="https://user-images.githubusercontent.com/1418580/165974116-3e9de941-e78b-4ec8-940e-a0647c619e9f.png">

Reply:
Any explanation why it renders that way? Is it a rendering bug?

Reply:
I think that is because `20)` is interpreted as a numbered list item starting at number 20.

Reply:
Ah, that makes a bit more sense then. A markdown gotcha.